lib/ex_doc/translation.ex

```diff
@@ -0,0 +1,569 @@
+        "You may want to try searching this site using the sidebar"
+      ),
+      do: "Quizás quieras intentar buscarla usando la barra lateral"
+
+  def t(@locale_spanish_international, "annotation"), do: "anotación"
+
+  def t(@locale_spanish_international, "anywhere (such as"),
+    do: "en cualquier lugar (tal como"
+
+  def t(@locale_spanish_international, "author"), do: "autor"
+  def t(@locale_spanish_international, "autocomplete"), do: "autocompletar"
+
+  def t(@locale_spanish_international, "before a word (such as"),
+    do: "antes de una palabra (tal como"
+
+  def t(@locale_spanish_international, "context"), do: "contexto"
+  def t(@locale_spanish_international, "contexts"), do: "contextos"
+  def t(@locale_spanish_international, "deprecated"), do: "obsoleto"
+  def t(@locale_spanish_international, "designed by"), do: "diseñado por"
+  def t(@locale_spanish_international, "detail"), do: "detalle"
+  def t(@locale_spanish_international, "details"), do: "detalles"
+  def t(@locale_spanish_international, "example"), do: "ejemplo"
+  def t(@locale_spanish_international, "examples"), do: "ejemplos"
+  def t(@locale_spanish_international, "exceptions"), do: "excepciones"
+  def t(@locale_spanish_international, "extras"), do: "extras"
+  def t(@locale_spanish_international, "footer"), do: "pie de página"
+  def t(@locale_spanish_international, "function"), do: "función"
+  def t(@locale_spanish_international, "functions"), do: "funciones"
+  def t(@locale_spanish_international, "group"), do: "grupo"
+  def t(@locale_spanish_international, "header"), do: "cabecera"
+  def t(@locale_spanish_international, "headers"), do: "cabeceras"
+  def t(@locale_spanish_international, "heading"), do: "encabezamiento"
+  def t(@locale_spanish_international, "help"), do: "ayuda"
+  def t(@locale_spanish_international, "is deprecated."), do: "está obsoleto."
+  def t(@locale_spanish_international, "list"), do: "lista"
+  def t(@locale_spanish_international, "link"), do: "enlace"
+  def t(@locale_spanish_international, "loading"), do: "cargando"
+  def t(@locale_spanish_international, "module"), do: "módulo"
+  def t(@locale_spanish_international, "modules"), do: "módulos"
+  def t(@locale_spanish_international, "name"), do: "nombre"
+  def t(@locale_spanish_international, "night mode"), do: "modo noche"
+  def t(@locale_spanish_international, "node"), do: "nodo"
+  def t(@locale_spanish_international, "nodes"), do: "nodos"
+  def t(@locale_spanish_international, "option"), do: "opción"
+  def t(@locale_spanish_international, "options"), do: "opciones"
+
+  def t(@locale_spanish_international, "or using our page"),
+    do: "o utilizar nuestra página"
+
+  def t(@locale_spanish_international, "project"), do: "proyecto"
+  def t(@locale_spanish_international, "search"), do: "buscar"
+  def t(@locale_spanish_international, "section"), do: "sección"
+
+  def t(@locale_spanish_international, "selected disabled"),
+    do: "seleccionado deshabilitado"
```

Comment:
where is this being used?

Reply:
It is only used in `assets/js/templates/versions-dropdown.handlebars`, but it is not currently being translated, so I guess should be removed from every translation.